tests/unit/common/test_environment.py

```
@@ -99,6 +100,40 @@ def test_validate_profile_configure_timeout(
+        settings = _APIServerSettings()
+        assert settings.HOST == "127.0.0.1"
+        assert settings.PORT is None
+        assert settings.CORS_ORIGINS == []
+
+    def test_api_server_settings_env_port_overrides_default(self, monkeypatch) -> None:
+        monkeypatch.setenv("AIPERF_API_SERVER_PORT", "8080")
+        settings = _APIServerSettings()
+        assert settings.PORT == 8080
+
+    def test_api_server_settings_env_host_overrides_default(self, monkeypatch) -> None:
+        monkeypatch.setenv("AIPERF_API_SERVER_HOST", "0.0.0.0")
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)